python/tk_multi_publish2/api/plugins/collector_instance.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2017 Shotgun Software Inc.
```

Comment:
I believe this is identical to https://github.com/shotgunsoftware/tk-multi-publish2/blob/master/python/tk_multi_publish2/processing/plugin.py#L410-L478